packages/webpack-configs/src/transformers/applyTransformers.ts

```diff
@@ -0,0 +1,13 @@
+import type { Configuration } from "webpack";
+
+// TODO: Might want to add stuff like the environment variables?
```

Comment:
2 TODOs here, what is the plan for those?

Reply:
Yeah, I expect to work on a PR tomorrow to support env variables.